scapy/config.py

```diff
@@ -690,7 +690,7 @@ class Conf(ConfClass):
     #: if 1, prevents any unwanted packet to go out (ARP, DNS, ...)
     stealth = "not implemented"
     #: selects the default output interface for srp() and sendp().
-    iface = Interceptor("iface", None, _iface_changer)
+    iface = Interceptor("iface", None, _iface_changer)  # type: 'scapy.interfaces.NetworkInterface'  # type: ignore  # noqa: E501
```

Comment:
I have not found a way to type `Interceptor` without an `ignore` tag. This is meta stuff too complex for Mypy